### PR TITLE
[IMP] stock_account: remove the wizard which change the ``standard_price``

### PR DESCRIPTION
**Pull request by std-odoo**
Overwrite the method ``write`` of the model ``product.product``.

When we write on ``standard_price``, if
- the context key ``disable_auto_svl`` is not set
- the ``cost_method`` is not "fifo"
We automatically compute ``stock.valuation.layer``

(before we needed to manually call ``_change_standard_price``)

Task #2031422

### CODE REVIEW

File: addons/product/views/product_template_views.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
                 <field name="company_id" options="{'no_create_edit': True}"
                     groups="base.group_multi_company" optional="hide"/>
                 <field name="list_price" string="Sales Price" optional="show"/>
-                <field name="standard_price" optional="show" readonly="1"/>
+                <field name="standard_price" optional="show"/>
```
Comment (sle-odoo) [inline addons/product/views/product_template_views.xml]: remove

File: addons/stock_account/models/product.py [hunk 2]
```diff
@@ -172,6 +182,10 @@ def _change_standard_price(self, new_price, counterpart_account_id=False):
         :param new_price: new standard price
         """
         # Handle stock valuation layers.
+
+        if self.valuation == 'real_time' and not self.env['account.move'].check_access_rights('create', raise_exception=False):
+            raise UserError(_('Cannot create change the price manually if the valuation mode is automatic and if you do not have the right to create accounting entries.'))
```
Comment (sle-odoo) [inline addons/stock_account/models/product.py]: to reword

Comment (sle-odoo) [reply addons/stock_account/models/product.py]: "An accounting entry has to be created and you don’t have the mandatory access rights."

Comment (sle-odoo) [reply addons/stock_account/models/product.py]: apparently oma encountered another one

oma@odoo.com: Sorry, you are not allowed to access documents of type 'Stock Valuation Layer' (stock.valuation.layer). This operation is allowed for the groups:      - Inventory/Administrator - (Operation: read, User: 6)

Comment (sle-odoo) [reply addons/stock_account/models/product.py]: "You cannot update the cost of a product in automated valuation as it leads to the creation of a journal entry, for which you don't have the access rights."

**Comment by sle-odoo** [comment]
robodoo r+

**Comment by sle-odoo** [comment]
robodoo retry

**Comment by d-fence** [comment]
robodoo retry
sorry @sle-odoo another shity runbot deploy

**Comment by sle-odoo** [comment]
robodoo retry

**Comment by sle-odoo** [comment]
robodoo retry

**Comment by sle-odoo** [comment]
aaaaarg

**Comment by d-fence** [comment]
@sle-odoo wiat a minute, we try to fix. We are deploying a new runbot version.

**Pull request closed without merging**